apps/console/src/features/connections/components/edit/forms/helpers/form-fields-helper.tsx

```diff
@@ -178,6 +178,48 @@ export const getRadioButtonField = (eachProp: CommonPluggableComponentPropertyIn
     );
 };
 
+// Add a radio field for user id in claims with predefined options.
+export const getUserIdClaimRadioButtonField = (eachProp: CommonPluggableComponentPropertyInterface,
+    propertyMetadata: CommonPluggableComponentMetaPropertyInterface,
+    listen: (key: string, values: Map<string, FormValue>) => void,
+    testId?: string): ReactElement => {
```

Comment:
We can improve formatting here.
```suggestion
export const getUserIdClaimRadioButtonField = (
    eachProp: CommonPluggableComponentPropertyInterface,
    propertyMetadata: CommonPluggableComponentMetaPropertyInterface,
    listen: (key: string, values: Map<string, FormValue>) => void,
    testId?: string
): ReactElement => {
```

Reply:
Improved with 5047722b4b25f0551dd0da7ca417f606a3ce039a.